Doc/whatsnew/3.14.rst

```diff
@@ -135,6 +135,12 @@ symtable
 
   (Contributed by Bénédikt Tran in :gh:`120029`.)
 
+socket
+------
+
+Add constants for IsoTp CAN protocol.
+(Contributed by Patrick Menschel and Stefan Tatschner in :issue:`42653`.)
```

Comment:
```suggestion
(Contributed by Patrick Menschel and Stefan Tatschner in :gh:`86819`.)
```

I think you can now use the GH issue instead of the old bpo.

Reply:
done